README.md

```diff
@@ -145,11 +145,11 @@ To test, you will use a Node.js script (`bin/msg.mjs`). Follow these steps:
 
 1. Create a user at the Cognito user pool while defining ther user's password:
 
-   `node bin/msg.mjs create-user -p <password>`
+   `node bin/msg.mjs create-user -u <username> -p <at-least-8-chars-password-with-1-number-and-1-spec-char>`
```

Comment:
I was thinking to make the instructions very simple. If you don't pass "-u", the command should create a user "testUser". Did you see any different behavior? The same would apply to the "auth" command.

Could you do another test to check if the situation was that there was an error with the password not following the restrictions, and then you needed to create another user because "testUser" was already created and in an invalid state? If this is the case, I think we should add a troubleshooting section.

Reply:
Yes, I'll have a look, that's the same thing you said... when it creates the user... it then sets a not conforming password in an already created user "testUser". So yes, we could add a troubleshoot section for deleting the user and passing the password